<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainManager</class>
 <widget class="QWidget" name="mainManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineSearch">
       <property name="placeholderText">
        <string>type service name to search...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterBox">
       <item>
        <property name="text">
         <string>Servers</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>System Services</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Startup Services</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Running Services</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All Services</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="listServices">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="progress">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="format">
      <string>Getting service info... %p%</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
